src/current/_includes/releases/v22.1/v22.1.21.md

```diff
@@ -0,0 +1,30 @@
+## v22.1.21
+
+Release Date: June 5, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-21-db-console-changes">DB Console changes</h3>
+
+- If a page crashes, a force refresh is no longer required to be able to see the other pages on DB Console. [#103326][#103326]
```

Comment:
suggest making 'DB Console' a link to https://www.cockroachlabs.com/docs/v22.1/ui-overview.html